File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.py
from collections import Counter

word1 = input().strip()
word2 = input().strip()

counter1 = Counter(word1)
counter2 = Counter(word2)

# 두 단어의 차이를 계산하여 최소 개수의 제거가 필요한 문자 수를 구함
remove = sum((counter1 - counter2).values()) + sum((counter2 - counter1).values())
print(remove)

File: 백준/Bronze/1919. 애너그램 만들기/애너그램 만들기.java
import java.util.*;

public class Main {   
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);

        String word1 = in.nextLine().trim();
        String word2 = in.nextLine().trim();
        
        Map<Character, Integer> frequencyMap = new HashMap<>();

        for (char ch : word1.toCharArray()) {
            frequencyMap.put(ch, frequencyMap.getOrDefault(ch, 0) + 1);
        }

        for (char ch : word2.toCharArray()) {
            frequencyMap.put(ch, frequencyMap.getOrDefault(ch, 0) - 1);
        }
        
        int answer = frequencyMap.values().stream()
            .mapToInt(Math::abs)
            .sum();
        
        System.out.println(answer);
    }

   
}